docs/pages/web/buttontoggle.tsx

```diff
@@ -0,0 +1,321 @@
+\`iconStart\` adds an icon before the ButtonToggle text.
+`}
+          title="Icons"
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={
+              <SandpackExample
+                // TODO: Fix this example to use iconStart instead of iconEnd
+                code={iconEndExample}
+                name="Icon start of buttontoggle example."
+                previewHeight={PREVIEW_HEIGHT}
+              />
+            }
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          columns={2}
+          description={`
+1. Default
+    The initial state of a ButtonToggle that represents it is in a non-selected state.
+2. Disabled
+Used to block user interaction such as hover, focus and click. Disabled ButtonToggles are completely unreachable by a keyboard and screenreader, so do not attach Tooltips to disabled ButtonToggles.
+3. Selected
+  When ButtonToggle is currently active or selected.
+`}
+          title="States"
+        >
+          <MainSection.Card
+            cardSize="md"
+            sandpackExample={
+              <SandpackExample
+                code={defaultStateExample}
+                name="Default state buttontoggle example."
+                previewHeight={150}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="md"
+            sandpackExample={
+              <SandpackExample
+                code={disabledStateExample}
+                name="Disabled state buttontoggle example."
+                previewHeight={150}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="md"
+            sandpackExample={
+              <SandpackExample
+                code={selectedStateExample}
+                name="Selected state buttontoggle example."
+                previewHeight={150}
+              />
+            }
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Writing">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            description={`
+- Use fewer than three words, ideally only one.
+- Use clear and concise copy for labels, tooltips, and any supporting text.
+- To make it clearer, you may want to change the label text to indicate that the ButtonToggle has been selected. For instance, changing "follow" to "Following."
+- Make sure that all text is easy to translate for localization purposes.
+`}
+            type="do"
+          />
+          <MainSection.Card
+            cardSize="md"
+            description={`
+- Do not use punctuation.
+`}
+            type="don't"
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <QualityChecklist component={generatedDocGen?.displayName} />
+
+      <MainSection name="Related">
+        <MainSection.Subsection
+          description={`
+**[Button](/web/button)**
```

Comment:
this related section is a big copy paste. 


ButtonToggle is a larger alternative to selection components such as [Checkbox](https://deploy-preview-3577--gestalt.netlify.app/android/checkbox), [RadioButton](https://gestalt.pinterest.systems/web/radiobutton), and [Switch](https://deploy-preview-3577--gestalt.netlify.app/android/switch).

I'd replace all this ones with Checkbox,RadioButton, and Switch

Reply:
DONE